packages/api/README.md

```diff
@@ -132,6 +180,9 @@ if (AppBskyFeedPost.isRecord(post)) {
 }
 ```
 
+Note that the agent class will validate records for you, so you don't need to do
+this before calling the API.
+
```

Comment:
I'm not sure that this is this case.  The `@atproto/xrpc` package doesn't validate params/inputs to my knowledge, and if it did then it wouldn't necessarily validate specific record types passed to e.g. `com.atproto.repo.createRecord`.

Reply:
Hum. Not sure why I wrote that. I guess I was daydreaming about my own view of the ideal behavior of the xrpc client's call implementation 😜